tests/__snapshots__/test_CodeFormer.ambr

```diff
@@ -4,8 +4,8 @@
     architecture='CodeFormer',
     input_channels=3,
     output_channels=3,
-    purpose='FaceSR',
-    scale=8,
+    purpose='Restoration',
```

Comment:
This is incorrect. Codeformer is meant to be run on faces and faces only. That's what the FaceSR category is for. This is true to the point where the only way to properly run this model is by using something like facexlib first to extract the face in the right way for the model to work properly, then stitch the result back in. The model is useless otherwise.

Reply:
Okay, thanks for the clarification! I'll change that back.